src/MyApplication.App/Program.cs

```diff
@@ -14,26 +20,95 @@ public static void Main(string[] args)
         {
```

Comment:
At least for all .net tutorials from Microsoft they use Top-Level-Statements now. Do we want to go one step further and use those for the template as well? 🤔

Reply:
Sure, let's go all the way.